src/os/bluestore/BlueStore.cc

```diff
@@ -6289,6 +6289,8 @@ void BlueStore::_txc_state_proc(TransContext *txc)
       txc->log_state_latency(logger, l_bluestore_state_prepare_lat);
       if (txc->ioc.has_pending_aios()) {
 	txc->state = TransContext::STATE_AIO_WAIT;
+	++txc->osr->txc_with_unstable_io;
+	txc->had_ios = true;
```

Comment:
I think that change is safe, but I'm not sure it would help: if the new txn is queuing new io, it is in teh queuing calling context, which means that there is no other txc currently in queue context (ios are not submitted in parallel within a sequencer)